Server/src/models/VariableModel.ts

```diff
@@ -0,0 +1,24 @@
+import { Datapoints } from "./entities/Datapoints";
+import { IVariableNameModel } from "../../../Client/src/Models/IVariableNameModel"
+import { getConnection, getManager } from "typeorm";
+
+
+export class VariableModel {
+  /**
+  * This method will return all existing variables or datapoints in the format of IVariableModel
+  */
+  async getAllVariables(): Promise<IVariableNameModel[]> {
+    let variables = await getManager()
```

Comment:
Solid query! Small request; change getManager() to getConnection() as that's what we use elsewhere

Reply:
Done